app/imports/components/Projects/CreateNewProject/CreateNewProject.jsx

```diff
@@ -1,26 +1,67 @@
 import React, { Component, PropTypes } from 'react';
 import { TextField,
     RaisedButton,
+    AutoComplete,
+    List,
+    ListItem,
 } from 'material-ui';
 
 class CreateNewProject extends Component {
 
     constructor(props) {
         super(props);
         this.onSubmit = this.onSubmit.bind(this);
+        this.addMods = this.addMods.bind(this);
         this.state = {
             isError: false,
             isResult: false,
+            usersNames: [],
+            usersIds: [],
+            chosenModerators: [],
+            chosenMenbers: [],
+            inputError: false,
+            memberError: false,
         };
     }
 
+    componentWillMount() {
+        const users = this.props.userList;
+
+        const names = [];
+        const ids = [];
+        for (let i = 0; i < users.length; i += 1) {
+            names.push(users[i].services.github.username);
```

Comment:
Profile name is optional, first time I`ve used profile.name and got undefinied
Also profiles names can be duplicated